modules/virt-vtpm-device-limitations.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/virt-using-vtpm-devices.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vtpm-device-limitations_{context}"]
+= Limitations of using vTPM devices
+
+With {VirtProductName} 4.18 and newer, you can export virtual machines (VMs) with attached vTPM devices, create snapshots of these VMs, and restore from them. However, cloning a VM with a vTPM device attached to it or creating a new VM from its snapshot is not supported.
```

Comment:
Clone not being supported may be relevant enough to be included in a note, wdyt? And maybe including links to the export and snapshot/restore APIs when referencing them is a good idea?

Reply:
Hi @alromeros, you are right. I removed the new section and moved that content in the admonition at the top of the vTPM section to ensure that this information cannot be missed, and linked to the docs for the mentioned operations:

- https://88581--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/managing_vms/virt-using-vtpm-devices.html

I then added a shorter note to the chapter about cloning VMs:

- https://88581--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/creating_vms_advanced/creating_vms_advanced_web/virt-cloning-vms.html

What do you think?

(I'll fix the vale bot complaints and failed pipelines.)